### ISSUE TITLE
Openssl on Ubuntu 18.04

### ISSUE THREAD

**Issue by dmaasland** [hashview/hashview#417]
Hi Guys,

Just installed this on Ubuntu 18.04, but was unable to do so initially due to OpenSSL 1.1. For future reference, this fixed the install for me on Ubuntu 18.04 (with RVM):

- apt-get install openssl1.0
- rvm install ruby-2.2.2 -- --with-openssl-dir=/usr/lib/ssl1.0/

*edit* hmm other problems now, might need some help with this one :):

```
RACK_ENV=production rake db:setup --trace
rake aborted!
ArgumentError: undefined class/module Digest::Class
/usr/local/rvm/rubies/ruby-2.2.2/lib/ruby/2.2.0/openssl.rb:17:in `require'
/usr/local/rvm/rubies/ruby-2.2.2/lib/ruby/2.2.0/openssl.rb:17:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/redis-3.3.2/lib/redis/connection/ruby.rb:8:in `require'
/usr/local/rvm/gems/ruby-2.2.2/gems/redis-3.3.2/lib/redis/connection/ruby.rb:8:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/redis-3.3.2/lib/redis/connection.rb:9:in `require'
/usr/local/rvm/gems/ruby-2.2.2/gems/redis-3.3.2/lib/redis/connection.rb:9:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/redis-3.3.2/lib/redis.rb:2775:in `require'
/usr/local/rvm/gems/ruby-2.2.2/gems/redis-3.3.2/lib/redis.rb:2775:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/redis-namespace-1.5.2/lib/redis/namespace.rb:1:in `require'
/usr/local/rvm/gems/ruby-2.2.2/gems/redis-namespace-1.5.2/lib/redis/namespace.rb:1:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-1.26.0/lib/resque.rb:2:in `require'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-1.26.0/lib/resque.rb:2:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-scheduler-4.3.0/lib/resque/scheduler/delaying_extensions.rb:2:in `require'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-scheduler-4.3.0/lib/resque/scheduler/delaying_extensions.rb:2:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-scheduler-4.3.0/lib/resque/scheduler/extension.rb:4:in `require_relative'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-scheduler-4.3.0/lib/resque/scheduler/extension.rb:4:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-scheduler-4.3.0/lib/resque-scheduler.rb:4:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-scheduler-4.3.0/lib/resque/scheduler/tasks.rb:4:in `require'
/usr/local/rvm/gems/ruby-2.2.2/gems/resque-scheduler-4.3.0/lib/resque/scheduler/tasks.rb:4:in `<top (required)>'
/home/user/tools/hashview/Rakefile:2:in `require'
/home/user/tools/hashview/Rakefile:2:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/rake_module.rb:28:in `load'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/rake_module.rb:28:in `load_rakefile'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/application.rb:686:in `raw_load_rakefile'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/application.rb:96:in `block in load_rakefile'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/application.rb:178:in `standard_exception_handling'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/application.rb:95:in `load_rakefile'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/application.rb:79:in `block in run'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/application.rb:178:in `standard_exception_handling'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/lib/rake/application.rb:77:in `run'
/usr/local/rvm/gems/ruby-2.2.2/gems/rake-11.2.2/exe/rake:27:in `<top (required)>'
/usr/local/rvm/gems/ruby-2.2.2/bin/rake:23:in `load'
/usr/local/rvm/gems/ruby-2.2.2/bin/rake:23:in `<main>'
/usr/local/rvm/gems/ruby-2.2.2/bin/ruby_executable_hooks:15:in `eval'
/usr/local/rvm/gems/ruby-2.2.2/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by dmaasland** [comment]
Small update on this, I tried again from scratch (removing everything, etc.). Turns out rvm already sorts the OpenSSL 1.1 vs 1.0 issue for you, so those commands seem to be useless (perhaps for use without RVM, don't know :)). 

However, when running "bundle install" it gave me a 403 error when downloading the "digest" gem (the same one that gave an error earlier). Turns out this gem is no longer available on rubygems for some reason. Commenting it out of the Gemfile seems to make everything work, however I can't really tell what the digest gem is used for.
